README.md

```diff
@@ -0,0 +1,58 @@
+# Qibo cloud backends
+
+For the complete documentation please refer to [qibo-cloud-backends](https://qibo.science/qibo-cloud-backends/stable/)
+
+## Installation instructions
+
+Install first the package dependencies with the following commands.
+
+We recommend to start with a fresh virtual environment to avoid dependencies
+conflicts with previously installed packages.
+
+``` bash
+   $ python -m venv ./env
+   source activate ./env/bin/activate
+```
```

Comment:
In principle, this part is not specific to this package, but general Python knowledge. To avoid repetition, you could even omit it.